CHANGELOG.md

```diff
@@ -106,3 +119,145 @@
 * YAML syntax file, improved inline syntax detection, automated building (thanks to matteo.campinoti94@gmail.com)
 * fixes typos in Description for AsciiDoc.forceUnixStyleSeparator (thanks to ccb012100)
 * diagram (specifically plantUML) support in asciidoctor.js (closes #105)
+
+# Changes
```

Comment:
Was this heading left here on purpose? Seems like an overlook.

Reply:
I will fix after merging, thanks for the look